src/Traits/DataModelTrait.php

```diff
@@ -172,4 +172,27 @@ public static function update_all( $set, $where = [] )
             ->where( $where )
             ->update();
     }
-}
\ No newline at end of file
+    
+    
+    /**
+     * Update data by ID
+     * @param int $id
+     * @param array $data
+     * @return bool 
+     */
+    public static function updateData($id, $data)
```

Comment:
We need to look for a more generic method name. Maybe `::modify()`. Also, needs to follow comments standard, same as the other methods. Unit testing is missing.